compiler-wasm/static/build/packages/packages.toml

```diff
@@ -0,0 +1,2 @@
+[packages]
+gleam_stdlib = "0.18.0"
```

Comment:
What's this file for?

Reply:
I can't recall why I created that file. Removed.